superset/assets/cypress_build.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
 set -e
 
-superset/bin/superset db upgrade
-superset/bin/superset load_test_users
-superset/bin/superset load_examples
-superset/bin/superset init
-flask run -p 8081 --with-threads --reload --debugger &
-
 cd "$(dirname "$0")"
 
-yarn install --frozen-lockfile
-npm run build
-npm run cypress run
+#run all the python steps in a background process
+(time /home/travis/build/apache/incubator-superset/superset/bin/superset db upgrade; time /home/travis/build/apache/incubator-superset/superset/bin/superset load_test_users; /home/travis/build/apache/incubator-superset/superset/bin/superset load_examples; time /home/travis/build/apache/incubator-superset/superset/bin/superset init; echo "[completed python build steps]"; flask run -p 8081 --with-threads --reload --debugger) &
+
+#block on the longer running javascript process
```

Comment:
It seems like we should have `flask run` run after we've run `npm run build` as then there's no need to background the task above.

Reply:
npm run build takes a really long time esp when it freezes on 92%. So I decided to put in as much as possible in the background while it is running so as to save the time it will take to run if it were to run after npm run build